### DISCUSSION TITLE
Option to disable email notifications

### DISCUSSION THREAD

**Discussion by watson20201** [Azure/api-management-developer-portal#2542] [Feature request]
Is it possible to disable the Welcome email being sent out when a user signs up?
Thanks

**Comment by azaslonov** [comment]
Hi @watson20201, this functionality isn't provided in API Management.

**Comment by mikebudzynski** [comment]
Extending this request to account for more notification types.

**Comment by laurynasr** [comment]
We have abstracted out the usage of subscription keys (they are still used, but our users don't directly use them or need to know about them), and the "subscription created" emails do not make any sense in our setup. There should be a way to control which notifications are sent and which are not.
If there is no way to disable those, then at least add ability to use subscription display name as a parameter for notification. The "New subscription activated" parameter list for some reason does not include that: 
![image](https://user-images.githubusercontent.com/18733763/94964110-e4671880-0501-11eb-8d31-b7cf8dc76752.png)

**Comment by campifoo** [comment]
Any workarounds for this?   We really need to disable notifications for a use case we have.  We have tried changing email source address and removing template (as much as possible) but emails still sent.

**Comment by mikebudzynski** [comment]
You should be able to disable notifications via network configuration, if your API Management service is in a virtual network.

**Comment by campifoo** [comment]
What network configuration do we need to do?  block smtp from tat virtual network?  we are on internal vnet.

**Comment by mikebudzynski** [comment]
Yes, I would start with that.

![image](https://user-images.githubusercontent.com/40186513/106200582-0c8f1d00-616c-11eb-9e57-91680df6c162.png)
More details here: https://docs.microsoft.com/en-us/azure/api-management/api-management-using-with-vnet

Please, test it in non-production services first and reach out to Azure Support if you have any questions.

**Comment by campifoo** [comment]
Thanks for getting back so quickly!!  Really helps.

**Comment by danmcpherson** [comment]
Any update on this one? Its a really critical bit of functionality because it's confusing for users to get these emails from someone that is not the organisation they signed up with. All the webhooks and tools are there to handle this via other approaches (logic apps). Going down the vNET route is really a heck, and introduces a lot of complexity just to get this simple outcome.

**Comment by hershize** [comment]
@azaslonov do you know when it will be available in API management? Either through the portal or via the management API?

**Comment by alexbenitez** [comment]
Is there any update regarding this limitation?
